plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -147,56 +187,142 @@ func (p *pusher) start() {
 		}
 	}()
 
-	for {
-		select {
-		case e := <-ec:
-			// Start timer when first event of the batch is added (happens after a flush timer timeout)
-			if len(p.events) == 0 {
-				p.resetFlushTimer()
-			}
+	if p.flagPersistentQueue {
+		for {
+			select {
+			case <-p.ticker.C:
+				//p.Log.Debugf("start dequeue from disk")
```

Comment:
If we don't need these debug logs, we can just remove them from the code. No need to comment them out